site/en/docs/extensions/reference/commands/index.md

```diff
@@ -110,3 +107,133 @@ Example:
   ...
 }
 ```
+
+## Examples
+
+The following examples show
+
+### Basic command
+
+Commands allow extensions to map logic to keyboard shortcuts that can be invoked by the user. At
+it's most basic, a command only requires a command declaration in the extension's manifest and a
+listener registration as shown below.
+
+```json
+// manifest.json
+{
+  "name": "Command demo - basic",
+  "version": "1.0",
+  "manifest_version": 3,
+  "background": {
+    "service_worker": "background.js"
+  },
+  "commands": {
+    "inject-script": {
+      "suggested_key": {
+        "default": "Ctrl+Shift+Y",
+        "mac": "Command+Shift+Y"
+      },
+      "description": "Inject a script on the page"
+    }
+  }
+}
+```
+
+```js
+// background.js
+chrome.commands.onCommand.addListener((command) => {
+  console.log(`Command "${command.name}" triggered`);
```

Comment:
Newcomers don't know where to see these messages and think "it doesn't work". A note somewhere with a gif/screenshot would be useful. Or maybe just add the permissions and use chrome.scripting.executeScript with a console.log inside so it'll be printed in the tab but of course many users will try it on a chrome page like chrome://extensions that can't be injected...

Reply:
I'm currently working on a debugging overview that will discuss this. I've opened #947 to remind me to link to that from this and other references.

Reply:
Good call, @toph. Thanks, for jumping on that, @awfuchs.